Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -960,6 +972,13 @@ bool WebProcessProxy::shouldAllowNonValidInjectedCode() const
 }
 #endif
 
+bool WebProcessProxy::isBlobRegistryPartitioningEnabled()
```

Comment:
It is never accurate to use ALL webpages to determine a setting when the setting is per session. This function should be taking a sessionID and only iterate through pages / processes with the given sessionID.